### PR TITLE
[BitSail][Connector] Support INSERT IGNORE INTO in mysql connector.

### PR DESCRIPTION
**Pull request by john8628**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests.

## Purpose

https://github.com/bytedance/bitsail/issues/142

Some description about what this PR wants to do.

## Approaches

Some description about how this PR achives the purpose. 

## Related Issues

_e.g._ Close #796

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

File: bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+package com.bytedance.bitsail.connector.legacy.jdbc.utils.ignore;
+
+import com.bytedance.bitsail.connector.legacy.jdbc.sink.JDBCOutputFormat;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MysqlInsertIgnoreUtil extends JDBCInsertIgnoreUtil {
```
Comment (garyli1019) [inline bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java:25]: this class seems not being used

File: bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  public MysqlInsertIgnoreUtil(JDBCOutputFormat jdbcOutputFormat, String[] shardKeys) {
+    super(jdbcOutputFormat, shardKeys);
+  }
+
+  @Override
+  public String genInsertIgnoreTemplate(String table, List<String> columns) {
```
Comment (garyli1019) [inline bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java:31]: that will be great if we can add unit test and functional test

**Review by garyli1019 — COMMENTED** [review]
@john8628 thanks for your contribution. left some comments

File: bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java (hunk 2 above)
Comment (john8628) [reply bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/utils/ignore/MysqlInsertIgnoreUtil.java:31]: Ok;thank you

**Review by garyli1019 — APPROVED** [review]
LGTM! Thanks for your contribution. For a newly added feature, we would like to change the doc to include this feature as well. The doc is located at `doc/connectors/jdbc/jdbc.md`(and chinese version if possible). I will merge this one first. Would you submit a follow up PR to change the doc? @john8628

**Comment by john8628** [comment]
> hello @john8628 , I found there are some checkstyle issue, would you please fix it?

ok ,thank you

**Pull request merged**